azkaban-common/src/main/java/azkaban/metrics/MetricsManager.java

```diff
@@ -28,24 +28,26 @@
 import java.lang.reflect.Constructor;
 import java.time.Duration;
 import java.util.concurrent.TimeUnit;
+import javax.inject.Inject;
```

Comment:
Definitely, I appreciate the feedback. This one kind of got away from me as I was dealing with unraveling the dependencies. I also didn't understand Guice very well going into it, so it was a lot of learning on the fly 😅